index.html

```diff
@@ -2372,7 +2392,10 @@ <h3>Algorithm</h3>
           <a href="#value-compaction">Value Compaction algorithm</a>,
           passing <var>active context</var>, <var>inverse context</var>,
           <var>active property</var>,and <var>element</var> as <var>value</var> is
-          a <a>scalar</a>, return that result.</li>
+          a <a>scalar</a>,
+          <span class="changed">or the <a>term definition</a> for <var>active property</var>
+            has a <a>type mapping</a> of <code>@json</code>,</span>
+          return that result.</li>
```

Comment:
"that result" is ambiguous here. It originally meant "the result of the *Value Compaction Algorithm*, but now the algorithm is not called in all cases.